src/Template/Bake/Element/form.ctp

```diff
@@ -19,9 +19,9 @@ $fields = collection($fields)
         return $schema->columnType($field) !== 'binary';
     });
 %>
-<div class="actions columns large-2 medium-3">
-    <h3><?= __('Actions') ?></h3>
+<nav class="large-3 medium-4 columns" id="actions-sidebar">
```

Comment:
Not depending on whatever-framework-was used style.
being able to replace the presentation.
http://alistapart.com/article/separationdilemma
http://alistapart.com/column/no-good-can-come-of-bad-code

In this case the way without a preprocessor would be to copy the class attribute to css as a comment also refering to the franework utilized AND below create the very few selectors and include their properties.

I am very happy with the inprovement though as now it looks good!